cfme/fixtures/v2v.py

```diff
@@ -20,15 +21,23 @@ def v2v_providers(request, second_provider, provider):
 @pytest.fixture(scope='function')
 def host_creds(request, v2v_providers):
     """Add credentials to conversation host"""
-    provider = v2v_providers[1]
-    hosts = provider.hosts.all()
     try:
+        vmware_provider = v2v_providers[0]
```

Comment:
The assumption that the vmware provider will be the 0th element and the rhv provider will be the second is a bit dangerous. Doesn't this depend solely on the order in which they are passed with the --use-provider option?

Reply:
I have not noticed any difference in provider order based on `--use-provider` because,  according to the code below, `second_provider` is always VMware and `v2v_providers` fixture returns the providers as 
`second_provider, provider` (VMware, RHV)

```
pytestmark = [
    pytest.mark.ignore_stream('5.8'),
    pytest.mark.provider(
        classes=[RHEVMProvider],
        selector=ONE_PER_VERSION,
        required_flags=['v2v']
    ),
    pytest.mark.provider(
        classes=[VMwareProvider],
        selector=ONE_PER_VERSION,
        fixture_name='second_provider',
        required_flags=['v2v']
    )
]
```

Reply:
Yep, that could be a trouble, I'll add checks there. :+1: